ark/attest/cache/snapshots.ts

```diff
@@ -206,7 +209,9 @@ const runFormatterIfAvailable = (queuedUpdates: QueuedUpdate[]) => {
 				)
 			)
 		]
-		shell(`${formatter} ${updatedPaths.join(" ")}`)
+		const command = formatter[0]
+		const args = formatter.slice(1)
+		shell(command, [...args, "--", ...updatedPaths])
```

Comment:
The extra `--` is to prevent a filename named like `--help` from being interpreted as a flag. Arguably needs to be applied elsewhere but it gets increasingly difficult elsewhere since shell commands rarely are prepared to actually deal with dashes at the beginning of filenames in arbitrary locations.